### PR TITLE
Add add to autostart btn, Update PyQt to v6

### PR DESCRIPTION
**Pull request by QWERTZexe**
Added an autostart button, which, when pressed, automatically inserts the batch file into the autostart.
Updated PyQt to v6

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -111,12 +116,32 @@ def load_task(self):
...
+
+                # Create a shortcut to the batch file in the Startup folder
+                shortcut_path = os.path.join(startup_folder, "TaskSaver.lnk")
+                with open(shortcut_path, 'w') as shortcut:
+                    shortcut.write(f'@echo off\n')
+                    shortcut.write(f'start "" "{batch_file_path}"\n')
+
+                QtWidgets.QMessageBox.information(self, "Success", "Task Manager has been added to autostart.")
+            except Exception as e:
+                QtWidgets.QMessageBox.critical(self, "Error", f"Error adding to autostart: {e}")
```
Comment (E-N1) [inline main.py:138]: Have you tried restarting your computer? What version of Windows do you have? In my opinion it doesn't work, but the approach is nice.

**Comment by E-N1** [comment]
Have you tried restarting your computer and the new feature works? What version of Windows are you running? On my computer, it doesn't work to start the program in Startup Programs. But the approach is nice and we can add to it

**Comment by QWERTZexe** [comment]
@E-N1 You were right. I reworked the function and tested it with both, windows 11 and windows 10. Let me know if it still doesnt work for you

**Comment by E-N1** [comment]
Nice, now it works 👍

**Pull request merged**